graylog2-web-interface/src/components/search/MessageFieldExtractorActions.jsx

```diff
@@ -34,15 +34,16 @@ class MessageFieldExtractorActions extends React.Component {
   };
 
   render() {
-    const messageField = this.props.message.fields[this.props.fieldName];
```

Comment:
Not really part of this PR, but could you move the `key` prop in the above `_formatExtractorMenuItem` method from the `MenuItem` to the `LinkContainer`? This will fix a warning. Otherwise I can do this in a separate PR.

Reply:
✅